app/src/main/java/com/yacgroup/yacguide/list_adapters/TourbookAscendViewAdapter.kt

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
I'm currently working on unifying all list adapters into one to reduce code duplication and split the code better into the responsible classes. This means, that this class will be removed again later.